src/function/gds/single_shortest_paths.cpp

```diff
@@ -16,25 +16,44 @@ using namespace kuzu::graph;
 namespace kuzu {
 namespace function {
 
+// TODO(Xiyang): Merge this class with ParentPtrsAtomics & PathMultiplicities
```

Comment:
Did this refactor sneak in? I wouldn't put this as part of this PR or update the PR comments.

Reply:
It's not in this PR. I'm just leaving a comment for myself. Will solve in the next couple of PRs.